base/loading.jl

```diff
@@ -3117,13 +3117,19 @@ end
             end
             for chi in includes
                 f, ftime_req = chi.filename, chi.mtime
-                if !ispath(f)
+                if ispath(f)
```

Comment:
Is it even necessary to branch on the value of `ispath(f)` here? Can we just get rid of the branch, and do `_f = fixup_stdlib_path(f)` unconditionally?

Presumably, if `f` is already a valid filesystem path, we would expect the `replace` inside `fixup_stdlib_path` to have no effect, right?

Reply:
Or are we trying to avoid doing the potentially expensive `fixup_stdlib_path(f)` if we don't need to?